Makefile

```diff
@@ -11,7 +11,9 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	targets/openshift/deps-gomod.mk \
 )
 
+GO_MOD_FLAGS = -mod=readonly
```

Comment:
@aguidirh - so no more vendor ? -  seems like it does not have any impact on the PROW jobs - nice

What happens now to the vendor folder - is it still needed ?

Reply:
Hi @lmzuccarelli,

With the changes in this PR the module will always download the dependencies from go proxy or directly from the git repo. If it is present in the cache (/go/pkg/mod/) it will retrieve from there.

Vendor folder it is not needed anymore.